Documentation/templates/plist/runtime-swift4.md

```diff
@@ -0,0 +1,53 @@
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File name | plist/runtime-swift4.stencil |
+| Invocation example | `swiftgen plist -t runtime-swift4 …` |
+| Language | Swift 4 |
+| Author | David Jennes |
+
+## When to use it
+
+- When you need to generate *Swift 4* code
+- Loads the data from the Plist file at runtime
+
+## Customization
+
+You can customize some elements of this template by overriding the following parameters when invoking `swiftgen` in the command line, using `--param <paramName>=<newValue>`
+
+| Parameter Name | Default Value | Description |
+| -------------- | ------------- | ----------- |
+| `documentName` | `Document` | Allows you to change the prefix of the generated `enum` for each document. |
```

Comment:
This is only used if the input file has multiple documents, right? Which can't be the case for JSON nor PLIST, but only for YAML? So not sure why this is included in the PLIST & JSON templates as well as their respective documentation? 

I guess it's included in the template for sake of consistency and similarity with YAML, and I'm not sure we should keep the multi-document logic for them but that could make sense and could be argued… but in the documentation this confuses the documentation and explanations more than anything